loadbalancer/algorithm.go

```diff
@@ -127,9 +129,9 @@ func withFadeIn(rnd *rand.Rand, ctx *routing.LBContext, notFadingIndexes []int,
 
 	switch a := algo.(type) {
 	case *roundRobin:
-		return shiftToRemaining(a.rnd, ctx, notFadingIndexes, a.fadingWeights, now)
+		return shiftToRemaining(a.rnd, ctx, notFadingIndexes, now)
```

Comment:
Maybe make `*rand.Rand` field name consistent in all algorithms?

Reply:
but that's just a nitpick

Reply:
Resolved. :-)